gui/panel_brainentropy.m

```diff
@@ -1041,6 +1090,7 @@ function UpdatePanel()
         choices = {'cMEM', 'wMEM', 'rMEM'};
         selected = [ctrl.jMEMdef.isSelected() ctrl.jMEMw.isSelected() ctrl.jMEMr.isSelected()];
         if ~any(selected)
+            ctrl.JPanelref.setPreferredSize(java_scaled('dimension', 600, 650));
```

Comment:
Since the panel is not created at each change.
At creation (when references are shown) is forced to be big enough to hold the options for the three MEM cases